configuration/driver/icedrv_step.F90

```diff
@@ -732,7 +814,12 @@ subroutine step_dyn_ridge (dt, ndtd)
       integer (kind=int_kind) :: &
          i,            & ! horizontal indices
          ntrcr,        & !
-         nbtrcr          !
+         nbtrcr,       & !
+         j               ! ice category index
+      
+      real (kind=dbl_kind) :: &
+         closing_area, & ! fractional area closed due to ice dynamics
+         aice_post_advection ! ice area after advecting ice to fill in closed area
```

Comment:
Are these variables used?

Reply:
No, they were for testing. I've removed them.